FinancialTracker/ViewControllers/RemindersTableViewController.swift

```diff
@@ -34,10 +34,11 @@ class RemindersTableViewController: UITableViewController {
         }
         
         for reminder in reminders {
-            if transfers[reminder.type.rawValue] == nil {
-                transfers[reminder.type.rawValue] = []
+            let transferType = reminder.type.rawValue
+            if transfers[transferType] == nil {
+                transfers[transferType] = []
             }
-            transfers[reminder.type.rawValue]?.append(reminder)
+            transfers[transferType]?.append(reminder)
         }
         
         self.transfers = transfers
```

Comment:
Line 47 `isEmpty()`, tiny bit of improvement:

```swift
 private func isEmpty() -> Bool {
        for value in transfers.values {
            guard value.isEmpty else {
                return false
            }
        }
        return true
    }
```

Also, you can convert it into a computed property.

Reply:
<img width="441" alt="Screenshot 2022-02-21 at 11 06 56" src="https://user-images.githubusercontent.com/18184558/154922980-47aad3c5-7915-4f91-8ab6-0e2fc7f98803.png">

Reply:
`isEmpty` as name is not great. What is empty?